.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
not needed? it uses gitignore by default

Reply:
cool